internal/formats/github/format.go

```diff
@@ -0,0 +1,30 @@
+package github
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/anchore/syft/syft/sbom"
+)
+
+const ID sbom.FormatID = "github-1-json"
```

Comment:
Does "1" represent the schema version here? Curious if this aligns with a named schema version they've provided

Reply:
It was intended to... but they are actually at version 0 🤦